### PR TITLE
Added custom placeholder for colored TPS

### PR DESCRIPTION
**Pull request by PrinceBunBun981**
What was mentioned on the Discord today but for specific server names, was also suggested we do this.

### CODE REVIEW

File: pages/Custom Placeholders.md [hunk 1]
```diff
@@ -748,6 +748,70 @@ To display the ping of the players on the tab list we use `player` as parameter:
 playerComponent: "${player name} ${colored_ping player}"
 ```
 
+#### The `${colored_tps}` Placeholder
+
+In this example we look at how we can add a color code to the ping depending on how good it is.
```
Comment (CodeCrafter47) [inline pages/Custom Placeholders.md]: This says ping.

File: pages/Custom Placeholders.md [hunk 2]
```diff
@@ -748,6 +748,70 @@ To display the ping of the players on the tab list we use `player` as parameter:
...
+The following table shows the desired color scheme:
+
+| Ping | Color |
```
Comment (CodeCrafter47) [inline pages/Custom Placeholders.md]: Here's another one.

File: pages/Custom Placeholders.md [hunk 3]
```diff
@@ -748,6 +748,70 @@ To display the ping of the players on the tab list we use `player` as parameter:
...
+| ---- | ----- |
+| Above 20 | Green |
+| 18 to 20 | Green |
+| 15 to 17 | Yellow |
+| 10 to 14 | Orange |
+| 0 to 9 | Red |
+
+We have five cases now, but using a single conditional placeholder we can only distinguish two cases.
+Just like in the Viewer Colored Ping placeholder, we can use multiple conditional placeholders to distinguish multiple.
```
Comment (CodeCrafter47) [inline pages/Custom Placeholders.md]: And here.

**Review by CodeCrafter47 — COMMENTED** [review]
Please edit or remove the text copied from the ping placeholder.

**Comment by PrinceBunBun981** [comment]
> Please edit or remove the text copied from the ping placeholder.

Removed, sorry about that oversight.

**Pull request merged**